Jellyfin.Api/Helpers/DynamicHlsHelper.cs

```diff
@@ -301,6 +307,13 @@ public class DynamicHlsHelper
             AppendPlaylist(builder, state, variantUrl, newBitrate, subtitleGroup);
         }
 
+        if (!isLiveStream && (state.VideoRequest?.EnableTrickplay).GetValueOrDefault(false))
```

Comment:
For consistency with the code base
```suggestion
        if (!isLiveStream && (state.VideoRequest?.EnableTrickplay ?? false))
```